### PR TITLE
add bootstrap-drawer v1.0.6 with npm auto-update

### LINKED ISSUE #7350: [Request] Add bootstrap-drawer
**Library name:** bootstrap-drawer
**Git repository url:** https://github.com/clineamb/bootstrap-drawer
**npm package url(optional):** https://www.npmjs.com/package/bootstrap-drawer
**License(s):** (See below)
**Official homepage:** https://github.com/clineamb/bootstrap-drawer
**Wanna say something? Leave message here:**
The package is published on npm: https://www.npmjs.com/package/bootstrap-drawer.

The licence is presented like this:

~ LICENCE START ~
Copyright (c) 2015, Caroline Amaba

Permission to use, copy, modify, and/or distribute this software for any purpose with or without fee is hereby granted, provided that the above copyright notice and this permission notice appear in all copies.

THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
~ LICENCE END ~

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #7350 
@x09326 please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/clineamb/bootstrap-drawer
 * Official website (optional, not the repository):https://clineamb.github.io/bootstrap-drawer/
 * NPM package url (optional): https://www.npmjs.com/package/bootstrap-drawer
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 57
   - Count of watchers: 7
   - Count of forks: 14
 * NPM download stats (optional):
   - Downloads in the last day: 4
   - Downloads in the last week: 45
   - Downloads in the last month: 220

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7350, cc @clineamb

### CODE REVIEW

**Comment by x09326** [comment]
I think [this](http://clineamb.github.io/bootstrap-drawer/) can be added as ```"homepage"```.

File: ajax/libs/bootstrap-drawer/1.0.5/js/drawer.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
+/* ========================================================================
+ * Bootstrap: drawer.js v1.0.3
```
Comment (x09326) [inline ajax/libs/bootstrap-drawer/1.0.5/js/drawer.js]: Um...
This line said the version is v1.0.3, but you need v1.0.5.
I think you need to confirm this with the author.

Comment (PeterDaveHello) [inline ajax/libs/bootstrap-drawer/1.0.5/js/drawer.js]: Just need to make sure it's the same as npm package.

**Comment by kennynaoh** [comment]
@x09326 
I have confirmed with author by https://github.com/clineamb/bootstrap-drawer/issues/13#
And updated the version to 1.0.6
Please help me review it again.
Thanks!

**Comment by x09326** [comment]
@kennynaoh 
I think you had better rebase it with the latest master.

**Comment by kennynaoh** [comment]
@x09326 
I have rebased to the latest master branch.
Please help me review it again.
Thanks!

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

**Pull request merged**